src/test/regress/spec/isolation_shard_rebalancer_progress.spec

```
@@ -171,7 +171,10 @@ step "s7-get-progress"
 		targetport,
 		target_shard_size,
 		progress,
-		operation_type
+		operation_type,
+		source_lsn >= target_lsn as lsn_sanity_check,
+		source_lsn > '0/0' as source_lsn_available,
+		target_lsn = '0/0' as target_lsn_available
```

column name suggested the opposite.
```suggestion
		target_lsn > '0/0' as target_lsn_available
```